src/main/resources/translations/program_strings_en.properties

```
@@ -399,51 +399,51 @@ udc.crbutton = Create new combination
+prefs.checknewversion = Check for new jExifToolGUI version when starting the program
 prefs.chkboxrawtitle = No RAW viewer selected
 prefs.chkboxrawtext = You can't set a this as default viewer without a RAW viewer configured.
-prefs.etlocatecanceltitle = Cancelled locate ExifTool
-prefs.etlocatecanceltext = You cancelled providing the location of exiftool.<br>I will leave the current preference for what it is.
+prefs.etlocatecanceltitle = Cancelled locating ExifTool
+prefs.etlocatecanceltext = You cancelled providing the location of ExifTool.<br>I will leave the current preference for what it is.
 prefs.etwrongtitle = Wrong executable
-prefs.etwrongtext = This is not the exiftool executable/binary.<br>I can't use it and will now exit jExifToolGUI.
-prefs.locateprefimgfolder = Locate preferred default image folder ...
-prefs.locaterawviewer = Locate your preferred RAW image viewer ...
+prefs.etwrongtext = This is not the ExifTool executable/binary.<br>I can't use it and will now exit jExifToolGUI.
+prefs.locateprefimgfolder = Locate preferred default image folder…
+prefs.locaterawviewer = Locate your preferred RAW image viewer…
 prefs.settingssaved = Settings saved
-prefs.applanguage = Preferred application language (requires restart)
+prefs.applanguage = Preferred program language (requires restarting the program)
 prefs.whichdialog = Preferred file dialog
-prefs.awtdialog = Awt File dialog
+prefs.awtdialog = AWT-file dialog
 prefs.jfilechooser = JFilechooser (default)
-prefs.dialogexplained = The JFileChooser is a far more flexible file dialog allowing "on the fly" file filters, like "images". "audio formats", "video formats", etcera \
-but looks a bit simple (and ugly to some).<br>The Awt filedialog looks more like a native Operating System file dialog, but is less flexible.
-prefs.showdecdegrees = Show coordinates in Decimal Degrees (Exiftool defaults to Deg Min Sec)
+prefs.dialogexplained = The JFileChooser is a far more flexible file dialog allowing "on the fly" file filters, like "images". "audio formats", "video formats", etc. \
+but looks a bit simple (and ugly to some).<br>The AWT file-dialog looks more like a native operating system file-dialog, but less flexible.
+prefs.showdecdegrees = Show coordinates in decimal degrees (ExifTool defaults to Deg Min Sec)
 prefs.loglevel = Log level
-prefs.logleveltext = Log levels range from Off to Error (lightest) to Info (average; default value) to Trace (log everything)
-prefs.logrestart = Requires restart of the application
+prefs.logleveltext = Log levels range from "Off" to "Error" (lightest) to "Info" (average; default value) to "Trace" (log everything)
+prefs.logrestart = Requires restarting the program
 prefs.useg1group = Use G1 group instead of G for viewing
 prefs.udfilefilter = User defined file filter:
-prefs.filterexample = example: jpg,cr3,raf,mp4,mie (comma separated)
+prefs.filterexample = example: jpg,cr3,raf,mp4,mie (comma-separated)
```

```suggestion
prefs.filterexample = example: JPG,CR3,RAF,MP4,MIE (comma-separated)
```
Not checked. Guessing.

---

I will commit this and test in the file dialog. If it works, I am OK with it. Otherwise I need to change it back. On Linux/Unix we do have  case-sensitivity. I do not know how that works for extension filters.